rampwf/utils/submission.py

```diff
@@ -337,12 +338,12 @@ def bag_submissions(problem, cv, y_train, y_test, predictions_valid_list,
         test_idx = ([valid_is for (train_is, valid_is) in cv]
                     if step == 'valid' else None)
         score_dict = {}
-        for score in score_type:
+        for score_type in score_types:
```

Comment:
I am against this change: the distinction between the plural and singular is so close that it is prone to error and to exchange them in the code. I am not against using the plural thought:


```python
for st in score_types:
```
would be less prone to error

Reply:
OK.